Marlin/Marlin_main.cpp

```diff
@@ -457,7 +457,7 @@ static int serial_count = 0;
 // Inactivity shutdown
 millis_t previous_cmd_ms = 0;
 static millis_t max_inactive_time = 0;
-static millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
+millis_t stepper_inactive_time = (DEFAULT_STEPPER_DEACTIVE_TIME) * 1000UL;
 
```

Comment:
Note that `M84 S0` disables the feature. If you want "immediate" disable, try `M84 S1`.